### PR TITLE
Clarify teleport-event-handler config

### PR DESCRIPTION
**Pull request by WilliamLoy**
Some users are using the example config before reading through the changes that need to be made to it. This PR moves the example config to come after the change instructions as well as adds examples for Splunk Enterprise vs. Splunk Cloud.

### CODE REVIEW

**Review by ptgott — CHANGES_REQUESTED** [review]

File: docs/pages/includes/plugins/finish-event-handler-config.mdx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <Tabs>
-<TabItem label="Linux server">
+<TabItem label="Linux server - Splunk Enterprise">
```
[the remark this answers is not in the document]
Comment (WilliamLoy) [reply docs/pages/includes/plugins/finish-event-handler-config.mdx:2]: @ptgott No I haven't. Do you have a guide for using it?

[the remark this answers is not in the document]
Comment (ptgott) [reply docs/pages/includes/plugins/finish-event-handler-config.mdx:2]: You can either:
- Use the preview link that our GHA workflows post to docs PRs (this isn't being posted to the current PR discussion, so I'm guessing you'll need to rebase this branch against `master` first)
- Follow the instructions here to review your changes on a local dev server: https://goteleport.com/docs/contributing/documentation/reviewing-docs/

**Comment by zmb3** [comment]
@WilliamLoy do you plan to clean this up or should we close it?

**Comment by WilliamLoy** [comment]
@ptgott were you able to get this converted to an issue?

**Comment by ptgott** [comment]
@WilliamLoy I've opened an issue at #38184

**Pull request closed without merging**